src/core/packet.h

```diff
@@ -307,6 +307,7 @@ QuicPktNumEncode(
     )
 {
     for (uint8_t i = 0; i < PacketNumberLength; i++) {
+        // cppcheck-suppress objectIndex
```

Comment:
What's up here?

Reply:
The warning is `The address of local variable 'PacketNumber' might be accessed at non-zero index. [objectIndex]`. 
Can also disable this globally?

Reply:
Probably.